terraform/TEMPLATES/calico_nodes.html

```diff
@@ -105,10 +105,12 @@ <h2>K8s Cluster Topology</h2>
       </div>
      </div>
 
-     <div class="container">
-      <input type="submit" value="Previous" class="btn btn--wide" name="button" onclick="saveInput()">
-      <input id="submit" type="submit" value="Next" class="btn btn--wide" name="button" onclick="saveInput()">
-   </div>
+         <div class="container">
+            <div style="display: flex; flex-direction: row-reverse; justify-content: flex-end; width: 100%; padding-right: 10px; padding-left: 0px; margin-right: auto; margin-left: auto;">
```

Comment:
A class in CSS would be good for this, so you don't need to copy or duplicate the style everywhere. :)

Reply:
This would help with code consistency incase someone else needs to create new pages or make changes to this style.
If it was in a class it would only need to be changed in one place instead of six.